extra-network/bandwhich/autobuild/defines

```diff
@@ -6,3 +6,4 @@ BUILDDEP="llvm rustc"
 
 USECLANG=1
 ABSPLITDBG=0
+NOCARGOAUDIT=1
```

Comment:
Please, do not use editors without newline. Also, please bump REL for this change.

Reply:
Fixed